src/views/Dashboard/Customer/Home/PaymentHistory/Details.vue

```diff
@@ -203,12 +208,10 @@ export default {
 
     async handleViewEtherscan() {
       const anchor = document.createElement("a")
-      const orderId = this.payment?.id
-      const { transaction_hash } = await fetchTxHashOrder(orderId)
 
       // NOTE: Use anchor tag with "noreferrer noopener" for security
       // eslint-disable-next-line camelcase
-      anchor.href = `https://rinkeby.etherscan.io/tx/${transaction_hash}`
+      anchor.href = `https://rinkeby.etherscan.io/tx/${this.txHash}`
```

Comment:
This is a `rinkeby` URL. Never hardcode this, use environment variables.

Reply:
Thank you, nice catch, will updated asap